packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -153,6 +156,24 @@ function Overview ({ className, onStatusChange }: Props): React.ReactElement<Pro
     </div>
   ), [filterOn, t]);
 
+  const connectMetamaskAccount = () => {
+    injectMetamaskPolkadotSnapProvider("kusama");
+  
+    (async ()=> {
```

Comment:
extract this to separate method "queryMetamask" with docs and put it below "queryLedger"